src/styles/onboard.css

```diff
@@ -69,8 +69,9 @@
   --onboard-border-radius-3: 8px;
 
   /* Z-index */
-  --onboard-modal-z-index: 1201;
-  --onboard-account-select-modal-z-index: 1201;
+  /* General modals are 998 because Keystone modal is 999 */
+  --onboard-modal-z-index: 997;
+  --onboard-account-select-modal-z-index: 997;
```

Comment:
Not sure this a good idea. The default modal zIndex in MUI is 1200.

Reply:
[I've set the modal `z-index` to `998`.](https://github.com/safe-global/web-core/blob/keystone/src/styles/theme.ts#L256)

Reply:
What about the header and sidebar?

Reply:
It seems like there are a lot of dependents. I look into whether Keystone can provide us with a CSS variable to alter this ourselves.